RedHatInsights/rhsm-subscriptions
# SWATCH-1014 Configure ActiveMQ

https://issues.redhat.com/browse/SWATCH-1014

Test steps:
Set SWATCH_JMS_PRODUCER_ENABLED=true in application.properties and then start the swatch-contract application. You will see something like below in logs indicating that it received the UMB message:
```2023-03-22 13:54:21,427 INFO  [com.red.swa.ContractUMBMessageConsumer] (pool-11-thread-1) {} {
  "action" : "contract-updated",
  "redHatSubscriptionNumber" : "12400374",
  "currentDimensions" : [ {
    "dimensionName" : "test_dim_1",
    "dimensionValue" : "5",
    "expirationDate" : "2023-02-15T00:00:00Z"
  }, {
    "dimensionName" : "test_dim_2",
    "dimensionValue" : "10",
    "expirationDate" : "2023-02-15T00:00:00Z"
  } ],
  "cloudIdentifiers" : {
    "awsCustomerId" : "896801664647",
    "productCode" : "1e1234el8qbwnqimt2wogc5n"
  }
}
```

Note: Since the Partner gateway have changed the structure of response currently, it won't pull any information from API.
